README.md

```diff
@@ -90,6 +90,11 @@ If there are changes, it does the following
 4. Pushes the local changes to remote using the branch configured in the `branch` input.
 5. Creates a pull request using the `title` and `body` inputs. If a pull request exists for the branch, it's checked out locally, rebased with `-XTheirs` and pushed with `--force` to update the pull request with the new changes.
 
+The actions outputs following properties:
+
+ - `pr-number` - number of created/updated PR
+ - `result` - `created` or `updated` depending the PR was created or only updated
+
```

Comment:
another result could also be "no change":
https://github.com/gr2m/create-or-update-pull-request-action/blob/65a6b01b19b69d7865fcf484340b06548fc02e64/index.js#L81-L88

Should we add `unchanged` to the list of possible results? And in case of `unchanged`, should we set `pr-number` if a pull request exists, but has not been updated?

Also could you rename `pr-number` to `pull-request-number`? I try to avoid abbreviations, `pr` might not be obvious for people new to GitHub.